lib/components/gzip.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const zlib = require('zlib')
```

Comment:
Is this dependency required on `package.json`?

Reply:
zlib is a built-in library, so we don't need to import it using `package.json`